shared/util/component-processor/src/jvmMain/kotlin/com/rudolfhladik/componentprocessor/content/ComponentPoetGenerator.kt

```diff
@@ -0,0 +1,65 @@
+package com.rudolfhladik.componentprocessor.content
+
+import com.google.devtools.ksp.processing.CodeGenerator
+import com.google.devtools.ksp.symbol.KSClassDeclaration
+import com.squareup.kotlinpoet.ClassName
+import com.squareup.kotlinpoet.CodeBlock
+import com.squareup.kotlinpoet.FileSpec
+import com.squareup.kotlinpoet.KModifier
+import com.squareup.kotlinpoet.PropertySpec
+import com.squareup.kotlinpoet.TypeSpec
+import com.squareup.kotlinpoet.ksp.toTypeName
+import com.squareup.kotlinpoet.ksp.writeTo
+
+class ComponentPoetGenerator { // TODO cant access classes from other modules
```

Comment:
I guess this is WIP version of generator using Kotlin Poet?

It appears to be unused, pls
- remove this class (we can add it later when finished)
- remove kotlin poet dependency